docs/installation/deployment/kubernetes.rst

```diff
@@ -0,0 +1,280 @@
+=======================
+Deploying on Kubernetes
+=======================
+
+.. epigraph::
+
+    `Kubernetes`_ is an open-source system for automating deployment,
+    scaling, and management of containerized applications.
+
+    -- kubernetes.io
+
+
+Kubernetes (K8s) is one of the main supported targets for deployment. We
+provide a fully automated deployment toolset suitable for DevOps engineers
+and/or system administrators.
+
+This documentation will sketch out the architecture, document the initial
+requirements and teach you how to deploy on K8s.
+
+This setup is tested against a Google Cloud Kubernetes cluster with 4vCPU and
+15G of memory.
```

Comment:
Link to n2 platform specs and possibly performance.

Reply:
Added